README.md

```diff
@@ -62,17 +54,12 @@ after ./autogen.sh, is running ./configure, make and make install.
 
 To build without SSL/TLS support, use ./configure --without-openssl
 
-
----------------------------------
-4. BUILDING ON OS/X WITH HOMEBREW
+4. Install macOS with Homebrew
```

Comment:
Please leave the 'Building on macOS with Homebrew' section intact, and add another 'Installing on macOS with Homebrew section beneath'.